actions.json

```diff
@@ -0,0 +1,22 @@
+{
+    "public_private": {
+        "name": "Move to public or private",
+        "command": "$YNH_APP_ID/scripts/actions/public_private",
+        "user": "root",
+        "cwd": "/etc/yunohost/apps/",
+        "accepted_return_codes": [0, 1, 2, 3],
+        "description": {
+            "en": "Change the public access of the app."
+        },
+        "arguments": [
+            {
+                "name": "is_public",
+                "type": "string",
```

Comment:
Is there any specific reason not to use the boolean type as in the manifest?

Reply:
Yes, the reason is that you'll have to use a command line to do the job, and putting yes or no is clearer than 1 or 0.

Reply:
Yes, but that's already the case for the installation...
And boolean are rightly "converted" when presented/requested in the yunohost-admin.
So if we'd better use "true" or "false" on the CLI, I think it should be up to the core to manage the transformation on the CLI (hence avoiding your heavy parameter testings :wink: )

Reply:
I'm thinking about the final user who will have to put 0 or 1 instead of yes or no.
Since there's no GUI so far, we can only used CLI. And I think it's would be better to have a yes or no (or y/n) instead of 1/0.

When there's the answer "Is it a public app ?:", it seems more natural to answer yes than 1

Reply:
Well, I'm actually envisaging that the moulinette evolves to manage "true"/"false" entries on the CLI instead of 0/1 :blush: 
cc @Psycojoker

Reply:
Still, using boolean answer isn't natural for users I think.
I agree to use them when it's through a GUI, because it's transparent for the user. But I rather prefer a "human" answer when the user has to write it.

Reply:
While I get your point, I just can't agree: the person that will use this CLI command (provided he/she could have used the GUI interface when available) will have installed by CLI... and typed 0/1 :blush: 
So as a matter of homogeneity/factorization, I'd rather have a global answer to this problem instead of managing this here "by hand" in every script.

Reply:
I understand your consideration for homogeneity, so how could we merge our 2 considerations ?
Should we add something like `(1/0) (1=yes; 0=no)` instead `(yes/no)` to keep it clear ?

Reply:
That could be a very good start!

Reply:
Ah, missed or wasn't available for the ping, sorry.

Without reading this discussion here I've made this PR that should solve your issue https://github.com/YunoHost/yunohost/pull/498

It changes the user UI but still uses 1/0 in the backend.

Reply:
(don't hesitate to ping me on IRC/wathever, I'm way more reactive there)

Reply:
PR has been merged https://github.com/YunoHost/yunohost/pull/498